tdconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "compilerOptions": {
+        "moduleResolution": "node",
+        "lib": [
+            "es5",
+            "dom"
+        ],
+        "allowSyntheticDefaultImports": true
+    },
+    "include": [
+        "lib/index.ts",
+        "lib/**/*.ts",
+        "build/docs/aws-crt-nodejs/lib/index.ts",
+        "build/docs/aws-crt-nodejs/lib/common/*.ts",
+        "build/docs/aws-crt-nodejs/lib/native/*.ts"
```

Comment:
Browser is intentionally missing here?

Reply:
Yes. We'll bifurcate later, but for now, just keep it consistent with just the node stuff.